release-team/role-handbooks/release-notes/editing-flow.md

```diff
@@ -0,0 +1,245 @@
+# Release Notes Editing Flow
+  - [Introduction](#introduction)
+  - [Motivation](#motivation)
+  - [Workflow Operation](#workflow-operation)
+    - [krel release-notes --fix](#krel-release-notes--fix)
+      - [Starting the interactive mode](#starting-the-interactive-mode)
+      - [Reviewing and editing the release notes](#reviewing-and-editing-the-release-notes)
+      - [Exiting the review loop](#exiting-the-review-loop)
+    - [Submitting changes back](#submitting-changes-back)
+    - [Re-running the workflow](#re-running-the-workflow)
+      - [Modified Release Notes](#modified-release-notes)
+
+
+## Introduction
+
+Ensuring the production of a document comprised of well written and 
+informative release notes is one of the responsibilities of the 
+Release Team each cycle. 
+
+Release Engineering has developed various tools to help the Team 
+complete its duties. One of them is `krel` - the [Kubernetes Release
```

Comment:
Should that be an em dash? It looks the same in a monospaced font — but it's a different character. Like the one I used in the last sentence.

```suggestion
complete its duties. One of them is `krel` — the [Kubernetes Release
```